src/handbook/development/ops/dedicated.md

```diff
@@ -0,0 +1,73 @@
+---
+navTitle: FlowFuse Dedicated
+---
+
+# FlowFuse Dedicated
+
+FlowFuse Dedicated is our product offering where we will host a dedicated instance
+of the platform for a customer.
+
+This is a guide for the setup and delivery of a dedicated instance.
+
+### Pre-deployment
+
+In order to create the dedicated instance, some information will be required from the customer.
+
+ - **A domain name to host the platform on**. The core platform (`forge.`), broker (`mqtt.`) and
+   hosted instances will be made available under this domain.
+   The customer will need to setup the DNS entry for this domain to point at the AWS Route53
+   end-point.
+ - **Choice of AWS region**. We default to `eu-west-1` but customers may want to choose one more
+   local to them. Not all AWS regions are equal and we may need to review their choice for suitability.
+ - **Capacity planning**. Whilst the platform will scale to meet demand, we want to sure we provision
+   it at a suitable level for their intended usage. Understanding their planned usage and growth rate
+   will help us pick the right node size for the cluster.
+ - **Team/Instance types**. We will default to creating equivalent Team and Instance types as
+   we use on FFCloud. If a customer has specific needs in this area, this should be identified prior
+   to deployment.
+ - **SSO Configuration**. As with FFCloud, if the customer plans to integrate with their SSO provider,
+   we will need to capture the necessary information. This can be done after the initial deployment.
+
+
+### Deployment
+
+An issue should be raised in the CloudProject repository using the Dedicated Checklist template
+and assigned to the member of the engineering/ops team who will do the setup.
+
+This checklist covers the follow items:
+
+1. Create a new AWS sub-account for each dedicated env.
+2. Create user accounts for the ops team
+2. Use Terraform to setup initial cluster
+3. Setup grafana monitoring - including all the necessary alerting
+4. Setup initial admin account - store details in 1Password
```

Comment:
```suggestion
4. Setup initial admin account - store details in 1Password (with restricted access)
```

Reply:
We'll probably want a dedicated vault for it. We can update the process once we have that in place.